### ISSUE TITLE
Start fixing accessibility and SEO issues

### ISSUE THREAD

**Issue by ktohalloran** [azavea/raster-vision-website2#43]
This issue is to complete the easy fixes for accessibility and SEO and come up with a plan for the more difficult tasks. These tasks include:
- [x] Make internal buttons (like those for the repeatable configurations section) buttons
- [x] Make the external buttons (like the "Explore documentation" button in "How it Works") links, not buttons
- [x] All images should have descriptive alt tags
- [x] Add description meta tag to the head tag
- [x] Any other issues listed [here](https://validator.w3.org/nu/?doc=https%3A%2F%2Fazavea.github.io%2Fraster-vision-website2%2F)

The more difficult task involves rearranging for a well-structured and logical dom. This issue includes exploring how this restructure would work with Tailwind, getting a sense of what will be involved, and writing another issue to complete it.

**Comment by ktohalloran** [comment]
Next steps for accessibility written as issue #44.
